tools/ios_osxcross.py

```diff
@@ -24,3 +24,5 @@ def generate(env):
         ]
     )
     env.Append(CCFLAGS=["-stdlib=libc++"])
+
+    env.Append(CPPDEFINES=["IOS_ENABLED", "UNIX_ENABLED", "COREAUDIO_ENABLED"])
```

Comment:
This is not needed, as this tool is included by `ios.py` (I think we should actually move the code in this tool to be the same `ios.py` file).
```suggestion
```